app/views/practices/overview.html.erb

```diff
@@ -39,6 +39,37 @@
               <%= f.text_area :tagline, class: "usa-textarea #{ @practice.errors[:tagline].any? ? 'usa-input--error' : '' } display-block practice-editor-tagline-textarea" %>
               <p class="usa-error-message <%= @practice.errors[:tagline].any? ? 'fas fa-exclamation-circle fon' : 'display-none' %>">&nbsp;<span class="font-family-sans"><%= show_errors(@practice, :tagline) %></span></p>
             </div>
+            <p class="margin-top-3"><span class="text-bold">Practice thumbnail:</span>&nbsp;</p>
+            <div class="grid-row margin-top-1">
+              <div class="grid-col-6">
+                <% if @practice.main_display_image.present? %>
+                  <%= image_tag @practice.main_display_image.url, alt: "practice thumbnail for #{@practice.name}", class: 'height-mobile practice-editor-impact-photo radius-md' %>
+                <% else %>
+                  <div class="bg-base-lightest position-relative radius-md" style="height: 300px; width: 350px;">
+                    <i class="fas fa-images fa-4x text-base-lighter no-impact-image"></i>
+                  </div>
+                <% end %>
+              </div>
+              <div class="grid-col-6">
+                <div class="margin-bottom-205 margin-left-3 margin-top-1">
+                  <p class="thumbnail-upload-text">Upload a clear photo that will be shown as the representative image for this practice. You can upload a .jpg, .jpeg, or .png file and the size limit is 1GB.</p>
+                </div>
+                <div class="grid-row margin-left-3 flex-align-start">
+                  <div class="grid-col-fill">
+                    <div class="">
+                      <%= f.file_field :main_display_image, class: "hidden-upload hidden-impact-photo-upload", accept: 'image/*' %>
+                      <%= f.label :main_display_image, 'Upload new photo', class: "upload-main-display-image-link" %>
+                    </div>
+                    <% if @practice.main_display_image.present? %>
+                      <div>
+                        <%= f.check_box :delete_main_display_image, { class: "usa-checkbox__input"}, 'true', 'false' %>
```

Comment:
⚠️ I deviated from the original design and followed a similar pattern in the “Origin” section, specifically for the Creators’ headshots, where a checkbox was used to remove the avatar. I did this because I was not sure how to remove the Practice Thumbnail image using a link. 🤔 
<img width="484" alt="Screen Shot 2020-03-05 at 10 05 10 AM" src="https://user-images.githubusercontent.com/20211771/76121390-25716b80-5fb9-11ea-8c04-796d7159879f.png">

Reply:
fine with me as long as becca knows/signed off :)

Reply:
ok becca told me there should not be a checkbox so I'm going to create an issue in our backlog making sure we are consistent with this! :)

Reply:
The new issue is [DM-1429](https://agile6.atlassian.net/secure/RapidBoard.jspa?rapidView=22&projectKey=DM&view=planning&selectedIssue=DM-1429&issueLimit=100)